Jenkinsfile

```diff
@@ -9,6 +9,7 @@ pipeline {
     parameters {
         string(defaultValue: '', name: 'major_version', description: "Major version of the docs to be built")
         string(defaultValue: '', name: 'minor_version', description: "Minor version of the docs to be built")
+        string(defaultValue: '', name: 'patch_version', description: "Patch version of the docs to be built")
```

Comment:
no patch version - we don't update docs for patches, just for major/minor releases